website/src/pages/404.js

```diff
@@ -0,0 +1,45 @@
+/** @jsx jsx */
+
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { jsx, Global } from '@emotion/core';
+import { globalStyles, gridSize } from '@arch-ui/theme';
+import { SkipNavContent } from '@reach/skip-nav';
+
+import Layout from '../templates/layout';
+import { Container } from '../components/Container';
+
+export default () => (
+  <Layout>
+    {() => (
+      <>
+        <Global styles={globalStyles} />
+        <NotFound />
+      </>
+    )}
+  </Layout>
+);
+
+const layoutGutter = gridSize * 4;
+
+const NotFound = () => (
+  <div>
+    <SkipNavContent />
+    <Container
+      css={{
+        paddingTop: layoutGutter,
+        paddingBottom: layoutGutter,
+      }}
+    >
+      <h1
+        css={{
+          marginTop: 0,
+          fontSize: ['2.4rem', '3.2rem'],
```

Comment:
Create a css fallback so it would generate `font-size: 2.4rem; font-size: 3.2rem`. Did you mean to wrap the object with `mq`/facepaint?

Reply:
Whoops, sorry @mitchellhamilton I merged before I saw this comment!

@jordanoverbye - can you make this change in a new PR?